src/bitbucket/BitbucketAPI.ts

```diff
@@ -42,7 +42,7 @@ export class BitbucketAPI {
     options: MergeOptions = {},
   ): Promise<MergePullRequestResult> => {
     const {
-      id: landRequestId,
+      requestId: landRequestId,
```

Comment:
This caused issues while debugging as land request status id was printed instead of land request id.